app/views/user_case_infos/_form.html.erb

```diff
@@ -63,7 +63,7 @@
     <%= f.label :case_end_date %><br>
     <%= f.datetime_select :case_end_date %>
   </div>
-   <div class="field">
+   <div class="field" hidden>
```

Comment:
Am I correct in assuming we want to pass this document ID along, but not display it, therefore it's hidden?

If so, if we can pull this in any other way (infer it from the model relationships maybe?) that would be preferable to hiding it (would reduce the footprint of things that could go wrong by a user maliciously editing a form by hand)